### PR TITLE
manage-survey-created

### PR DESCRIPTION
**Pull request by nguyenhathanh2212**
templates table survey
redmine: https://dev.framgia.com/redmine/issues/167139

image

image
image

### CODE REVIEW

File: config/datatables.php [hunk 1]
```diff
@@ -0,0 +1,106 @@
+<?php
+
+return [
+    /**
+     * DataTables search options.
+     */
+    'search'          => [
```
Comment (lamngockhuong) [inline config/datatables.php]: convention

File: resources/assets/templates/survey/js/datatables-script.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
+$(document).ready(function() {
+    var locale = $('#list-survey').attr('locale');
+    var share = $('#share-language').attr('share');
+    var language = '';
+    var dataUrl = $('#list-survey').attr('data-url');
+
+    if (locale === 'vn') {
```
Comment (lamngockhuong) [inline resources/assets/templates/survey/js/datatables-script.js]: Fix i18n vào file lang nè :D

File: resources/views/survey/survey/list-survey.blade.php [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+            <div class="col-xl-12 push-xl-12 col-lg-12 push-lg-12 col-md-12 col-sm-12 col-xs-12">
+                <div class="ui-block">
+                    <div class="ui-block-content table-custom-survey">
+                        <table id="list-survey" class="stripe display hover cell-border"
+                            locale="{{ Session::get('locale') }}" width="100%" cellspacing="0"
+                            data-url="{{ route('survey.survey.get-surveys') }}">
+                            <thead>
+                                <tr>
+                                    <tr>
+                                        <th width=5%>@lang('lang.index')</th>
```
Comment (lamngockhuong) [inline resources/views/survey/survey/list-survey.blade.php]: Convention

File: app/Http/Controllers/Survey/SurveyManagementController.php [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+    public function showSurveys()
+    {
+        try {
+            $user = Auth::user();
+            Session::put('page_profile_active', config('settings.page_profile_active.list_survey'));
+
+            return view('survey.survey.list-survey', compact('user'));
+        } catch (Exception $e) {
+            return redirect()->route('home');
```
Comment (trangkarol) [inline app/Http/Controllers/Survey/SurveyManagementController.php]: xử lý đưa về 404

File: resources/assets/templates/survey/js/datatables-script.js [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+            {
+                aTargets: [0],
+                searchable: false,
+                orderable: false,
+                mRender: function (data, type, row, meta) {
+                    return meta.row + meta.settings._iDisplayStart + 1;
+                }
+            },
+            {
+                aTargets: [ 2 ],
```
Comment (lamngockhuong) [inline resources/assets/templates/survey/js/datatables-script.js]: Convention :D

File: resources/assets/templates/survey/js/datatables-script.js [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+                    var icon = full.status ? "<i class='fa fa-check'></i> " :
+                    "<i class='fa fa-times'></i> ";
+
+                    return "<span style='color: " + color + "'>"
+                    + icon + status_custom
+                    + "</span>";
+                }
+            },
+            {
+                aTargets: [ 3 ],
```
Comment (lamngockhuong) [inline resources/assets/templates/survey/js/datatables-script.js]: Convention

File: resources/assets/templates/survey/js/datatables-script.js [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+                searchable: false,
+                mRender: function(status_custom, type, full) {
+                    return "<a class='share-facebook' href=''>"
+                        + "<i class='fa fa-facebook-square'></i> "
+                        + share + "</a>";
+                }
+            },
+            {
+                aTargets: [ 4 ],
```
Comment (lamngockhuong) [inline resources/assets/templates/survey/js/datatables-script.js]: Convention

File: resources/assets/templates/survey/js/datatables-script.js [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+                sType: 'date',
+                mRender: function(date, type, full) {
+                    return date;
+                }
+            },
+        ],
+        order: [ 4, 'desc' ],
```
Comment (lamngockhuong) [inline resources/assets/templates/survey/js/datatables-script.js]: convention

File: resources/assets/templates/survey/js/datatables-script.js [hunk 9]
```diff
@@ -0,0 +1,85 @@
...
+
+    $('#list-survey').DataTable({
+        processing: true,
+        serverSide: true,
+        responsive: true,
+        ajax: {
+            url: dataUrl,
+        },
+        columns: [
+            { data: '' },
```
Comment (lamngockhuong) [inline resources/assets/templates/survey/js/datatables-script.js]: CoNvEnTiOn :D

File: resources/lang/jp/lang.php [hunk 10]
```diff
@@ -132,4 +132,38 @@
...
+    'sInfoEmpty' => ' 0 件中 0 から 0 まで表示',
+    'sInfoFiltered' => '（全 _MAX_ 件より抽出）',
+    'sInfoPostFix' => '',
+    'sInfoThousands' => ',',
+    'sLengthMenu' => '_MENU_ 件表示',
+    'sLoadingRecords' => '読み込み中...',
+    'sProcessing' => '処理中...',
+    'sSearch' => '検索:',
+    'sZeroRecords' => '一致するレコードがありません',
+    'sFirst' =>    '先頭',
```
Comment (lamngockhuong) [inline resources/lang/jp/lang.php]: Convention

**Pull request merged**